documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -0,0 +1,338 @@
+   radix_sort_by_key (sycl::queue q, KeysRng&& keys,
+                      ValuesRng&& values, KernelParam param); // (3)
+
+   template <bool IsAscending = true, std::uint8_t RadixBits = 8,
+             typename KernelParam, typename Iterator1, typename Iterator2>
+   sycl::event
+   radix_sort_by_key (sycl::queue q, Iterator1 keys_first, Iterator1 keys_last,
+                      Iterator2 values_first, KernelParam param); // (4)
+
+   }
+
+
+.. _template-parameters:
+
+Template Parameters
+--------------------
+
++-----------------------------+---------------------------------------------------------------------------------------+
+| Name                        | Description                                                                           |
++=============================+=======================================================================================+
+| ``bool IsAscending``        | The sort order. Ascending: ``true``; Descending: ``false``.                           |
++-----------------------------+---------------------------------------------------------------------------------------+
+| ``std::uint8_t RadixBits``  | The number of bits to sort per a radix sort algorithm pass.                           |
++-----------------------------+---------------------------------------------------------------------------------------+
+
+
+.. _parameters:
+
+Parameters
+----------
+
++-----------------------------------------------+---------------------------------------------------------------------+
+| Name                                          | Description                                                         |
++===============================================+=====================================================================+
+|  ``q``                                        | SYCL* queue to submit the kernels to.                               |
++-----------------------------------------------+---------------------------------------------------------------------+
+|                                               | The sequences of elements to apply the algorithm to.                |
+|  - ``r`` (1)                                  | Supported sequence types:                                           |
+|  - ``first``, ``last`` (2)                    |                                                                     |
+|  - ``keys``, ``values`` (3)                   | - ``sycl::buffer`` (1,3),                                           |
+|  - ``keys_first``, ``keys_last``,             | - ``oneapi::dpl::experimental::ranges::views::all`` (1,3),          |
+|    ``values_first`` (4)                       | - ``oneapi::dpl::experimental::ranges::views::subrange`` (1,3),     |
+|                                               | - USM pointers (2,4),                                               |
+|                                               | - ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` (2,4).            |
```

Comment:
For USM pointers and `begin()`, `end()`, we should link to the corresponding section on the "Pass Data to Algorithms" page.

Reply:
I've done it. I also referenced a section with oneDPL's viewable ranges.